acct-group/opendkim/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>gentoo@seichter.de</email>
+		<name>Ralph Seichter</name>
+	</maintainer>
+	<maintainer type="person">
+		<email>mjo@gentoo.org</email>
```

Comment:
Here's the real question for now though: Do you have his consent on being added here? Have you talked about it? Or would you like proxy-maint to be added?

Reply:
Yeah, Michael and I discussed this. If you check mail-filter/opendkim contribution history, you'll find Michael did significant work, and is also already listed in metadata.xml. I would not mind adding proxy-maint as well, but I thought you guys don't accept new packages anymore? In any case, I don't expect changes in acct-* ebuilds anyway.

Reply:
We do "accept" them, but currently dont have time to check those due to self-maintained queue. Acct- packages are necessary to keep self-maintaining already committed ebuilds, so they are merged. 

Anyway, @orlitzky wanna take over? You seem to have a personal interest over this.

Reply:
I can get to it eventually, but the two new packages look OK (they're just boilerplate) and the opendkim ebuild Ralph and I have already beaten to death, so everything's probably fine.

Reply:
Indeed. This is ebuild _r17_ after all. I only made modifications because of an auto-generated QA bug anyway.